deepchecks/vision/utils/transformations.py

```diff
@@ -118,24 +144,50 @@ def get_robustness_augmentations(cls, data_dim: Optional[int] = 3):
         return augmentations
 
 
-def get_transforms_handler(transforms):
+class TorchTransformations(AlbumentationsTransformations):
+    """Class containing supporting functions for torch transforms."""
+
+    @classmethod
+    def add_augmentation_in_start(cls, aug, transforms: T.Compose):
+        """Add given transformations to the start of given transforms object."""
+        if isinstance(aug, (albumentations.Compose, albumentations.BasicTransform)):
+            alb_aug = aug
+
+            class TorchWrapper:
```

Comment:
how this handle transformations on the label?

Reply:
torch.transforms does not transform labels to the best of my knowledge, so makes sense we won't have this option when using an albumentation aug inside a torch pipeline...